components/introduction.tsx

```
@@ -1,160 +1,114 @@
-          text: '보안에 특화된 엔터프라이즈 소프트웨어 기업',
-        },
-        {
-          id: 2,
-          url: 'https://jandi.com',
-          text: '업무용 협업툴 잔디(JANDI)를 서비스하는 SaaS 스타트업',
-        },
-      ],
-      blog: '블로그',
-      button: ['이메일', '링크드인', '깃헙', 'X(구 트위터)'],
-    },
+  const [content, setContent] = useState<any>(null)
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)